package.json

```diff
@@ -27,17 +27,21 @@
     "posttest": "yarn run lint"
   },
   "engines": {
-    "node": ">=10.10.0"
+    "node": ">=10.14.2"
```

Comment:
There were a couple of packages that wouldn't install at lower versions than this.